gpkit/tests/t_model.py

```diff
@@ -350,7 +328,7 @@ def test_signomials_not_allowed_in_objective(self):
             x = Variable('x')
             y = Variable('y')
             J = 0.01*((x - 1)**2 + (y - 1)**2) + (x*y - 1)**2
-            m = Model(J)
+            m = Model(J, [J >= 1])
```

Comment:
but replacing the `[J >= 1]` with `[]` causes the test to fail, which concerns me.